src/components/SideDrawer/FieldWrapper.tsx

```diff
@@ -126,11 +136,16 @@ export default function FieldWrapper({
           >
             {debugText}
           </Typography>
-
           <IconButton
-            href={`https://console.firebase.google.com/project/${projectId}/firestore/data/~2F${(
-              debugText as string
-            ).replace(/\//g, "~2F")}`}
+            onClick={() => {
+              copyToClipboard(documentPath);
```

Comment:
This button should copy the document path, not the URL to the Firebase Console.

Reply:
Done